hackathon_site/event/jinja2/event/login.html

```diff
@@ -1,10 +1,11 @@
 {% extends "event/form_base.html" %}
 
+{% block title %}Login - MakeUofT 2021{% endblock %}
 {% block form_title %}Login{% endblock %}
 
 {% block nav_logout_login_links %}{% endblock %}
 
-{% block form_img %}<img src="{{ static('event/images/login.svg') }}" alt="Login image"/>{% endblock %}
+{% block form_img %}<object type="image/svg+xml" data="{{ static('event/images/login.svg') }}" alt="Login image"></object>{% endblock %}
```

Comment:
The object tag doesn't appear to make images scale like `img` did, so on mobile this doesn't shrink down:

![image](https://user-images.githubusercontent.com/26036279/103159957-b9593180-479d-11eb-876b-198bbe45efcf.png)

Same with the rest of the images I assume

Reply:
fixed in styles.css